### PR TITLE
adding actual server

### PR DESCRIPTION
**Pull request by floco**
as discussed on discord

### CODE REVIEW

File: Apps/actual-server/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  architectures: # Supported CPU architectures
+    - amd64
+    - arm64
+  main: app # Main service of the application
+  description:
+    en_us: Actual Budget is a super fast and privacy-focused app for managing your finances. At its heart is the well proven and much loved Envelope Budgeting methodology. You own your data and can do whatever you want with it. Featuring multi-device sync, optional end-to-end encryption and so much more. # Description in English
+  tagline:
+    en_us: Actual Server # Short description or tagline in English
+  developer: "" # Developer's name or identifier
+  author: BigBearTechWorld # Author of this configuration
```
Comment (dragonfire1119) [inline Apps/actual-server/docker-compose.yml:26]: This would be your name or BigBearCommunity. :)

File: Apps/actual-server/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+# Service definitions for the big-bear-actual-server application
+services:
+  # Service name: app
+  app:
+    image: actualbudget/actual-server:latest # The Docker image to use (latest version of actual-server)
+    container_name: actual-server # Name of the container instance
+    volumes:
+      - /DATA/AppData/$AppID:/data # Mounting the project directory inside the container
+    ports:
+      - "5006:5006" # Expose actual-server's port 8080 on the host's port 8080
```
Comment (dragonfire1119) [inline Apps/actual-server/docker-compose.yml:12]: The comment need to be: 
```
Expose actual-server's port 5006 on the host's port 5006?
```

**Comment by dragonfire1119** [comment]
Also the README needs to be updated with the new app.

**Comment by floco** [comment]
all good points. should be better now.

**Pull request merged**